compound/src/main/kotlin/io/element/android/compound/tokens/generated/SemanticColorsDark.kt

```diff
@@ -0,0 +1,105 @@
+    bgCriticalSubtle = DarkColorTokens.colorRed200,
+    bgCriticalSubtleHovered = DarkColorTokens.colorRed300,
+    bgDecorative1 = DarkColorTokens.colorLime300,
+    bgDecorative2 = DarkColorTokens.colorCyan300,
+    bgDecorative3 = DarkColorTokens.colorFuchsia300,
+    bgDecorative4 = DarkColorTokens.colorPurple300,
+    bgDecorative5 = DarkColorTokens.colorPink300,
+    bgDecorative6 = DarkColorTokens.colorOrange300,
+    bgInfoSubtle = DarkColorTokens.colorBlue200,
+    bgSubtlePrimary = DarkColorTokens.colorGray400,
+    bgSubtleSecondary = DarkColorTokens.colorGray300,
+    bgSuccessSubtle = DarkColorTokens.colorGreen200,
+    borderCriticalHovered = DarkColorTokens.colorRed1000,
+    borderCriticalPrimary = DarkColorTokens.colorRed900,
+    borderCriticalSubtle = DarkColorTokens.colorRed500,
+    borderDisabled = DarkColorTokens.colorGray500,
+    borderFocused = DarkColorTokens.colorBlue900,
+    borderInfoSubtle = DarkColorTokens.colorBlue500,
+    borderInteractiveHovered = DarkColorTokens.colorGray1100,
+    borderInteractivePrimary = DarkColorTokens.colorGray800,
+    borderInteractiveSecondary = DarkColorTokens.colorGray600,
+    borderSuccessSubtle = DarkColorTokens.colorGreen500,
+    iconAccentPrimary = DarkColorTokens.colorGreen900,
+    iconAccentTertiary = DarkColorTokens.colorGreen800,
+    iconCriticalPrimary = DarkColorTokens.colorRed900,
+    iconDisabled = DarkColorTokens.colorGray700,
+    iconInfoPrimary = DarkColorTokens.colorBlue900,
+    iconOnSolidPrimary = DarkColorTokens.colorThemeBg,
+    iconPrimary = DarkColorTokens.colorGray1400,
+    iconPrimaryAlpha = DarkColorTokens.colorAlphaGray1400,
+    iconQuaternary = DarkColorTokens.colorGray700,
+    iconQuaternaryAlpha = DarkColorTokens.colorAlphaGray700,
+    iconSecondary = DarkColorTokens.colorGray900,
+    iconSecondaryAlpha = DarkColorTokens.colorAlphaGray900,
+    iconSuccessPrimary = DarkColorTokens.colorGreen900,
+    iconTertiary = DarkColorTokens.colorGray800,
+    iconTertiaryAlpha = DarkColorTokens.colorAlphaGray800,
+    textActionAccent = DarkColorTokens.colorGreen900,
+    textActionPrimary = DarkColorTokens.colorGray1400,
+    textCriticalPrimary = DarkColorTokens.colorRed900,
+    textDecorative1 = DarkColorTokens.colorLime1100,
+    textDecorative2 = DarkColorTokens.colorCyan1100,
+    textDecorative3 = DarkColorTokens.colorFuchsia1100,
+    textDecorative4 = DarkColorTokens.colorPurple1100,
+    textDecorative5 = DarkColorTokens.colorPink1100,
+    textDecorative6 = DarkColorTokens.colorOrange1100,
+    textDisabled = DarkColorTokens.colorGray800,
+    textInfoPrimary = DarkColorTokens.colorBlue900,
+    textLinkExternal = DarkColorTokens.colorBlue900,
+    textOnSolidPrimary = DarkColorTokens.colorThemeBg,
+    textPlaceholder = DarkColorTokens.colorGray800,
+    textPrimary = DarkColorTokens.colorGray1400,
+    textSecondary = DarkColorTokens.colorGray900,
+    textSuccessPrimary = DarkColorTokens.colorGreen900,
+    isLight = false,
```

Comment:
I have checked and this match what we had before, even better it fixes issue we had on `textDecorative*` 🤞